lib/iris/plot.py

```diff
@@ -801,10 +801,15 @@ def outline(cube, coords=None):
         element is the horizontal axis of the plot and the second element is
         the vertical axis of the plot.
 
+    * color: any matplotlib color.
+
+    * linewidth: float value for width of outline.
```

Comment:
This combines the type of the argument with the meaning of the argument. The preferred form these days is more like:
```
    * color: None or mpl color
        The color of the cell outlines. If None, the rc setting (which one???) is used by default.

    * linewidth: None or number
        The width of the lines showing the cell outlines. If None, the default width in lines.linewidth(???) in matplotlibrc is used.
```

NB. This could do with a more consistent description of the behaviour when None. (Or we just ignore it... :expressionless: )